### PR TITLE
re #1180: Add new video source : IntersonArraySDKCxx

### PR DESCRIPTION
**Pull request by mjallais**

### CODE REVIEW

File: src/PlusDataCollection/IntersonArraySDKCxx/vtkPlusIntersonArraySDKCxxVideoSource.h [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  class vtkInternal;
+  vtkInternal* Internal;
+
+  vtkPlusUsImagingParameters* ImagingParameters;
+
+private:
+  vtkPlusIntersonArraySDKCxxVideoSource(const vtkPlusIntersonArraySDKCxxVideoSource&);  // Not implemented.
+  void operator=(const vtkPlusIntersonArraySDKCxxVideoSource&);  // Not implemented.
+
+  unsigned char PulseVoltage;
```
Comment (adamrankin) [inline src/PlusDataCollection/IntersonArraySDKCxx/vtkPlusIntersonArraySDKCxxVideoSource.h]: Have these as protected please.

File: src/PlusLibConfig.cmake.in [hunk 2]
```diff
@@ -61,6 +61,15 @@ IF(PLUS_USE_INTERSONSDKCXX_VIDEO)
   FIND_PACKAGE(IntersonSDKCxx REQUIRED)
 ENDIF()
 
+# Get the location for IntersonArrayCxx
+SET(PLUS_USE_INTERSONARRAYSDKCXX_VIDEO "@PLUS_USE_INTERSONARRAYSDKCXX_VIDEO@")
```
Comment (adamrankin) [inline src/PlusLibConfig.cmake.in]: This Set is not needed, as it is done in UsePlusLib.cmake.in

Comment (adamrankin) [inline src/PlusLibConfig.cmake.in]: Ah, I see the other IntersonSDKCxx does this as well. If you're feeling generous, could you remove the other one as well?

**Comment by mjallais** [comment]
@adamrankin : IntersonSDK also has the variables PulseVoltage and RfDecimation as private. Should I move them as protected too ? I am feeling generous :)

**Comment by adamrankin** [comment]
Yes please!

**Comment by adamrankin** [comment]
Sorry about the delay, looks great!

**Pull request merged**